dobotapi/dobot.py

```diff
@@ -65,7 +65,7 @@ def _disconnect(self) -> None:
     def close(self) -> None:
         "Exits the dobot program properly"
 
-        self.gripper.close()
+        self.gripper.idle()
         self.suction_cup.idle()
         self.conveyor_belt.move(0)
```

Comment:
The `close()` needs work. Even though `self.gripper.idle()` returns the gripper to idle, the `self.suction_cup.idle()` closes it back up. It works if we switch the order of the lines but I'm not sure how it affects the suction cup. I'm only testing it with the gripper.

Perhaps we can track with addons are attached?

Reply:
or we can just take out statements for the addons in `close()`